generator/app.go

```diff
@@ -170,6 +255,7 @@ type Group struct {
 	Meetings         []Meeting
 	Contact          Contact
 	Subprojects      []Subproject `yaml:",omitempty"`
+	KEPs             map[string][]KEP
```

Comment:
In order to not populate the `sigs.yaml` with an empty dictionary, you can do this.
```suggestion
	KEPs             map[string][]KEP  `yaml:",omitempty"`
```

Reply:
Thanks for pointing. Resolved in the refreshed commit.